test/v1r6/services/AmendBenefitAmountsServiceSpec.scala

```diff
@@ -87,19 +88,19 @@ class AmendBenefitAmountsServiceSpec extends ServiceSpec {
 
       input.foreach(args => (serviceError _).tupled(args))
 
-      def serviceErrorSpaced(desErrorCode: String, error: MtdError): Unit =
-        s"a $desErrorCode error with empty spaces is returned from the service" in new Test {
+      def serviceErrorSpaced(ifsErrorCode: String, error: MtdError): Unit =
+        s"a $ifsErrorCode error with empty spaces is returned from the service" in new Test {
 
           MockUpdateBenefitAmountsConnector.updateBenefitAmounts(requestData)
-            .returns(Future.successful(Left(ResponseWrapper(correlationId, DesErrors.single(DesErrorCode(desErrorCode))))))
+            .returns(Future.successful(Left(ResponseWrapper(correlationId, DesErrors.single(DesErrorCode(ifsErrorCode))))))
 
           await(service.updateBenefitAmounts(requestData)) shouldBe Left(ErrorWrapper(correlationId, error))
         }
 
       val inputSpaced = Seq(
         ("INVALID_TAXABLE_ENTITY_ID ", NinoFormatError),
         (" INVALID_TAX_YEAR ", TaxYearFormatError),
-        ("INVALID_BENEFIT_ID ", NotFoundError),
+        ("INVALID_BENEFIT_ID ", BenefitIdFormatError),
```

Comment:
looks like there's still a space after `INVALID_BENEFIT_ID` here

Reply:
Ye thats for the ifsErrorCode test.  I think they've all got spaces in that Seq